renku/core/management/repository.py

```diff
@@ -58,7 +58,7 @@ def _path_converter(path) -> Path:
 class PathMixin:
     """Define a default path attribute."""
 
-    path: Path = attr.ib(default=default_path, converter=_path_converter)
+    path: Path = attr.ib(default=default_path, converter=_path_converter)  # type: ignore
```

Comment:
I fought with this before (we really should get rid of attr, this PR is a great start).

no typing on `_path_converter(path):` caused mypy to complain, doing `def _path_converter(path:Union[str,Path]) -> Path:` caused it to complain as well, but just specifying the output type, but not the input git mypy to somehow infer the type correctly. But i think it just doesn't work well with attrs. I have `attrs==21.4.0` locally, I'm curious which version you have. That might play a role as well, Maybe we need to pin attrs to a higher version?

In any case, I'm fine with just ignoring, just wanted to give some background info on this.